docs/general/how-to-define-semantic-conventions.md

```diff
@@ -0,0 +1,151 @@
+check out [common attributes](/docs/general/attributes.md).
+Semantic conventions authors are encouraged to use attributes from different namespaces.
+
+Consider adding a new attribute if all of the following apply:
+
+- It provides a clear benefit to end users by enhancing telemetry.
+- There is a clear plan to use the attributes when defining spans, metrics, events, resources, or other telemetry signals in semantic conventions.
+- There is a clear plan on how these attributes will be used by instrumentations
+
+Semantic convention maintainers may reject the addition of a new attribute if its benefits
+and use-cases are not yet clear.
+
+When defining a new attribute:
+
+- Follow the [naming guidance](/docs/general/naming.md)
+- Provide descriptive `brief` and `note` sections to clearly explain what the attribute represents.
+  - If the attribute represents a common concept documented externally, include relevant links.
+    For example, always link to attributes related to concepts defined in RFCs or other standards.
+  - If the attribute's value might contain PII or other sensitive information, explicitly call this out in
+    the `note`.
+
+    Include a warning similar to the following: <!-- TODO: update existing semconv -->
+
+    ```yaml
+      - id: user.full_name
+        ...
+        note: |
+          ...
+
+          > [!WARNING]
+          >
+          > This attribute contains sensitive (PII) information.
+    ```
+
+- Use the appropriate [attribute type](https://github.com/open-telemetry/weaver/blob/main/schemas/semconv-syntax.md#type)
+  - If the value has a reasonably short (open or closed) set of possible values, define it as an enum.
+  - If the value is a timestamp, record it as a string in ISO 8601 format.
+  - For arrays of primitives, use the array type. Avoid recording arrays as a single string.
+  - Use the template type to define attributes with variable names (only the last segment of the name should be dynamic).
+    This is useful for capturing user-defined key-value pairs, such as HTTP headers.
+  - Represent complex values as a set of flat attributes. <!-- This may change, check out https://github.com/open-telemetry/semantic-conventions/issues/1669 to monitor the progress -->
+- Define new attributes with `development` stability.
+- Provide realistic examples
+- Avoid defining attributes with potentially unbounded values, such as strings longer than
+  1 KB or arrays with more than 1,000 elements. Such values should be recorded in the log or event body instead.
+
+Consider the scope of the attribute and how it may evolve in the future:
+
+- When defining an attribute for a narrow use case, think about potential broader use cases.
+  For example, if creating a system-specific attribute, evaluate whether other systems
+  in the same domain might need a similar attribute in the future.
+
+  Similarly, instead of defining a simple boolean flag like `foo.is_error`, consider a
```

Comment:
is the `is_*` convention used for boolean flags?

Reply:
removed - I don't think we have a convention - it's an anti-example anyway